ideascube/serveradmin/views.py

```diff
@@ -167,3 +170,41 @@ def wifi_history(request):
     return render(
         request, 'serveradmin/wifi_history.html',
         {'wifi_list': wifi_list.values()})
+
+
+@staff_member_required
+def packages(request):
+    # XXX: Should have a local Kinto cache.
+    client = Client(server_url='http://kinto.bsf-intranet.org/v1',
+                    bucket='ideascube', collection='packages')
+    packages = client.get_records()
+
+    return render(
+        request, 'serveradmin/packages.html',
+        {'packages': packages})
+
+
+@staff_member_required
+def package_detail(request, package_id):
+    client = Client(server_url='http://kinto.bsf-intranet.org/v1',
```

Comment:
Note that you cannot install a remote package while offline, so this is a limitation of the feature.
But yeah, it definitely brings the usecase for offline first kinto.py :) I opened  about this https://github.com/Kinto/kinto.py/issues/25